cmd/podmanV2/pods/ps.go

```diff
@@ -19,14 +30,140 @@ var (
 	}
 )
 
+var (
+	defaultHeaders string = "POD ID\tNAME\tSTATUS\tCREATED"
+	inputFilters   string
+	noTrunc        bool
+	psInput        entities.PodPSOptions
+)
+
 func init() {
 	registry.Commands = append(registry.Commands, registry.CliCommand{
 		Mode:    []entities.EngineMode{entities.ABIMode, entities.TunnelMode},
 		Command: psCmd,
 		Parent:  podCmd,
 	})
+	flags := psCmd.Flags()
+	flags.BoolVar(&psInput.CtrNames, "ctr-names", false, "Display the container names")
+	flags.BoolVar(&psInput.CtrIds, "ctr-ids", false, "Display the container UUIDs. If no-trunc is not set they will be truncated")
+	flags.BoolVar(&psInput.CtrStatus, "ctr-status", false, "Display the container status")
+	// TODO should we make this a [] ?
+	flags.StringVarP(&inputFilters, "filter", "f", "", "Filter output based on conditions given")
```

Comment:
This definitely seems like it should be a StringArray

Reply:
today it is a comma sep string, which kind of sucks ... this will be a breaking change for pod ps but i think it is worth it

Reply:
I think we allow `,` in labels, so we can't filter on labels like this.

Reply:
@rhatdan mind weighing in here too? ok with the change?